### DISCUSSION TITLE
enable wayland on spiralinux

### DISCUSSION THREAD

**Discussion by geoubuntu** [SpiralLinux/SpiralLinux-project#324] [Q&A]
Hi. 
why is wayland not enabled on spiralinux gnome (using an intel card)?
I enabled it changing the gdm daemon.conf file but since then bluetooth service is not enabled at login anymore not even with systemctl enable bluetooth comand

**Comment by geckolinux** [comment]
https://github.com/orgs/SpiralLinux/discussions/316#discussioncomment-7215164
